mava/configs/arch/sebulba.yaml

```diff
@@ -0,0 +1,26 @@
+# --- Sebulba config ---
+architecture_name: sebulba
+
+# --- Training ---
+num_envs: 32  # number of environments per thread.
+
+# --- Evaluation ---
+evaluation_greedy: False # Evaluate the policy greedily. If True the policy will select
+  # an action which corresponds to the greatest logit. If false, the policy will sample
+  # from the logits.
+num_eval_episodes: 200 # Number of episodes to evaluate per evaluation.
+num_evaluation: 200 # Number of evenly spaced evaluations to perform during training.
+num_absolute_metric_eval_episodes: 32  # Number of episodes to evaluate the absolute metric (the final evaluation).
+absolute_metric: True # Whether the absolute metric should be computed. For more details
+# on the absolute metric please see: https://arxiv.org/abs/2209.10485
+
+# --- Sebulba devices config ---
+n_threads_per_executor: 2  # num of different threads/env batches per actor
+actor_device_ids: [0] # ids of actor devices
+learner_device_ids: [0] # ids of learner devices
+n_learner_accumulate: 1  # Number of envoirnments to accumulate before updating the parameters. This determines the num_envs for learning updates which equals (num_envs * n_learner_accumulate) / len(learner_device_ids).
```

Comment:
Ye as discussed offline I'm not 100% sold on this...even though it was my idea :joy: 

@Louay-Ben-nessir what do you think should we keep it and document it well or remove it and make the code simpler?

Reply:
I don't really know how useful this can be in the future but it's pretty simple under the hood so if this comment doesn't clear it up I'll just remove it and we can add it back if it's ever needed

# Number of environment accumulations required before the learner thread updates the model parameters. 
# Each accumulation consists of `num_envs` trajectories. The effective 
# number of environments used per learning device update is calculated as 
# (num_envs * n_learner_accumulate) / len(learner_device_ids).
# This parameter helps avoid small batch sizes when running the learner, 
# which can occur if there are is  small number of environments to learner devices ratio.

Reply:
SInce we are unsure whether this will be useful and we are not using it at the moment, I think we should remove it. The code is already quite complex so any simplification could be very useful.